setup.py

```diff
@@ -68,6 +70,7 @@ def __init__(self, *args, **kwargs) -> None:
     package_data={"raiden": ["py.typed"]},
     license="MIT",
     zip_safe=False,
+    long_description_content_type="text/markdown",
```

Comment:
Ref: https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata